src/d3a/d3a_core/rq_job_handler.py

```diff
@@ -96,6 +96,10 @@ def launch_simulation_from_rq_job(scenario, settings, events, aggregator_device_
                 ConstSettings.IAASettings.MARKET_TYPE = 2
                 ConstSettings.IAASettings.BID_OFFER_MATCH_TYPE = \
                     d3a.constants.BidOfferMatchAlgoEnum.PAY_AS_CLEAR.value
+            if spot_market_type == 4:
```

Comment:
Where can I find documentation about the `4`?

Reply:
Found it! https://github.com/gridsingularity/d3a-web/blob/41bcce49f00dedf83723ac868f0e393ac141236a/simulation/graphql/graphql_enums.py#L28-L31

I really think we should turn these integers into Enums.